src/serializable/primitives/bytes.ts

```diff
@@ -34,6 +34,10 @@ export class Bytes extends Primitives {
     return concatBytes(bytesForInt(this.bytes.length), this.bytes);
   }
 
+  toBytesWithoutLength() {
+    return this.bytes;
+  }
+
```

Comment:
should this be `value()`? That would seem to better align with other primitives maybe.